app/questionnaire/questionnaire_manager.py

```diff
@@ -125,3 +126,44 @@ def get_schema_item_by_id(self, item_id):
 
     def get_schema(self):
         return self._schema
+
+    def add_answer(self, block, question, post_data, answer_store):
```

Comment:
It would be clearer to rename `block` to `block_id` and `question` to `question_id` to make it clear they are id's and not objects